paper.md

```diff
@@ -226,9 +226,9 @@ Interoperability with scientific or commercial services and hosting services all
 Git can interact with other repositories on the file system or accessible via a set of standard (ssh, http) or custom (Git) network transport protocols.
 DataLad implements support for interactions with non Git-aware but commonly used portals via custom Git transfer protocols, as, e.g., it was done for interactions with the Open Science Framework (OSF) in the DataLad extension ``datalad-osf`` [@datalad-osf:zenodo].
 Git-annex provides access to a wide range of external data storage resources via various protocols but cannot implement all idiosyncrasies of any individual data portal.
-In particular, scientific data is frequently stored in compressed archives to reduce its storage demands, and/or on specialized servers, such as XNAT ([www.xnat.org](http://www.xnat.org)).
-To address these demands, git-annex established a protocol [@git-annex:special_remotes_protocol] through which external tools can provide custom transport functionality transparently to the git-annex user.
-This allowed DataLad and many other projects to facilitate access to an ever-growing collection of resources [@git-annex:special_remotes] and to overcome technological limitations (e.g., maximal file sizes, or file system inode limits).
+In particular, scientific data is frequently stored in compressed archives to reduce its storage demands, and/or on specialized servers, such as XNAT ([www.xnat.org](http://www.xnat.org)). Efficient scientific data processing therefore usually requires seamless access to a wide variety of different stores of scientific data.
```

Comment:
Just to follow consistent formatting (a sentence per line) I will reformat into two lines. 
I will go trimming slightly further: "to reduce its storage demands" might not be the only or not even the major reason why they do it -- might be the only way to provide a sane way to download multiple files at once, to minimize transfer traffic, to minimize time it might take to download millions of tiny files individually, etc:

```suggestion
In particular, scientific data is frequently stored in compressed archives, and/or on specialized servers, such as XNAT ([www.xnat.org](http://www.xnat.org)).
Efficient scientific data processing therefore usually requires seamless access to a wide variety of different stores of scientific data.
```